src/game_api/script/usertypes/entities_items_lua.cpp

```diff
@@ -311,8 +311,8 @@ void register_usertypes(sol::state& lua)
         &Spark::rotation_center_x,
         "rotation_center_y",
         &Spark::rotation_center_y,
-        "angle",
-        &Spark::angle,
+        "rotation_angle",
```

Comment:
Is this an API break? Previously when accessing `.angle` would that refer to `Entity.angle` or `Spark.angle`?

Reply:
just tested with overlunky, it's Spark.angle

to be honest, is soo stupid that VS won't mark this as an error, i can see myself spending hours to find that kind of mistake